src/event.js

```diff
@@ -347,11 +347,11 @@ jQuery.event = {
 			delegateCount = handlers.delegateCount,
 			cur = event.target;
 
-		// Support (at least): Chrome, IE9
+		// Support (at least): IE 9
```

Comment:
It would be valuable, but you don't _need_ to. Support comments document what we know, not what we don't.